fs/procfs/fs_procfsproc.c

```diff
@@ -762,6 +762,7 @@ static ssize_t proc_critmon(FAR struct proc_file_s *procfile,
                             size_t buflen, off_t offset)
 {
   struct timespec maxtime;
```

Comment:
why not rename `maxtime` to `ts` and reuse it instead of adding new stack variable?